### DISCUSSION TITLE
Meta: Using github labels and milestone

### DISCUSSION THREAD

**Discussion by jtxa** [sierrafoxtrot/srecord#30] [Ideas]
Milestones:
Don't think it's really needed for planning, but i might be beneficial for closed issues and PRs.

Labels:
Might help for a better overview on open issues.
Especially `bug` would also be useful for filtering, even on closed ones.
There is a predefined set of labels, so they could be immediately used.

**Comment by sierrafoxtrot** [comment]
The labels like bug definitely make sense and while I'd love to keep the issues list small, the filtering is always helpful.

> Milestones: Don't think it's really needed for planning, but i might be beneficial for closed issues and PRs.

Interesting. How would it help with closed issues and PRs? Are you thinking in terms of knowing when it's a good time to cut a release?

**Reply by jtxa** [reply]
As a user landing on a PR/issue after searching the internet, it's always nice to know, in which release that thing is solved/included. So you immediately know if this is relevant for the version you use or not. Most Linux distributions will contain a different version, not latest.

P.S. For PRs you can simply search, e.g. `is:pr is:merged no:milestone` and simply mass edit them when doing the release.
On issues though, it would be manual work, as you cannot simply assume that all closed ones are part of a release.
My suggestion was based on having a benefit with minimal effort. So it's totally ok to just mark PRs and probably bug issues.

**Comment by jtxa** [comment]
About releases I haven't thought much yet. A strict planning won't be needed, I assume you will have the right feeling when it's time to do release.
Looking at the past, it was mostly 2-4 per year. 2007-2009 were the most active ones, with 8-10 releases.

**Reply by sierrafoxtrot** [reply]
> About releases I haven't thought much yet. A strict planning won't be needed, I assume you will have the right feeling when it's time to do release. Looking at the past, it was mostly 2-4 per year. 2007-2009 were the most active ones, with 8-10 releases.

Yeah, those were busy years with new formats being churned out from memory. It really does come down to gut feel usually with new formats/features, bugfixes being a good trigger. The usual stuff I guess. 

Merging the OS65A format from @chapmajs would definitely be a great point as well (fingers are crossed for that one soon!)

I'm working away on some additional CRCs which would be nice but mostly doing that out of personal interest rather than any request so it's a bit of a slow burn
